browser/brave_vpn/dns/brave_vpn_dns_observer_service_win.cc

```diff
@@ -9,11 +9,11 @@
 
 #include "base/strings/string_util.h"
 #include "brave/browser/ui/views/brave_vpn/brave_vpn_dns_settings_notificiation_dialog_view.h"
-#include "brave/components/brave_vpn/browser/connection/ikev2/win/brave_vpn_helper/brave_vpn_helper_constants.h"
-#include "brave/components/brave_vpn/browser/connection/ikev2/win/brave_vpn_helper/brave_vpn_helper_state.h"
 #include "brave/components/brave_vpn/common/brave_vpn_utils.h"
 #include "brave/components/brave_vpn/common/pref_names.h"
 #include "brave/components/brave_vpn/common/win/utils.h"
+#include "brave/installer/win/util/brave_vpn_helper_constants.h"
```

Comment:
what is used here from these files?

Reply:
both constants and utils

Reply:
From utils, the `Is*Running` and `Is*Installed` are being used. For example:
`IsBraveVPNHelperServiceInstalled`, `IsWindowsServiceRunning`, `IsNetworkFiltersInstalled`, `IsNetworkFiltersInstalled`. Some of the other help methods are used too (`GetBraveVpnHelperServiceName`, etc).

The include for constants can be removed - I'll remove that

Reply:
Removed in https://github.com/brave/brave-core/pull/20754/commits/03cb04a583f99b4d8530bbf8c6000344ad7f2a05